daemon/pod/container.go

```diff
@@ -1196,7 +1202,9 @@ func (cs *ContainerStatus) Start() error {
 	cs.Lock()
 	defer cs.Unlock()
 
-	if cs.State != S_CONTAINER_CREATED {
+	if cs.State == S_CONTAINER_RUNNING {
+		return errors.ErrContainerAlreadyRunning
+	} else if cs.State != S_CONTAINER_CREATED {
 		return fmt.Errorf("only CREATING container could be set to creatd, current: %d", cs.State)
```

Comment:
Seems this is a wrong error message, could you also fix it here.
I think this should be `only CREATED container could be set to RUNNING`